src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
-
+  shape: 'triangle'|'circle'|'rectangle';
+  color: 'red'|'green'|'blue';
+  getArea():number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle'|'circle'|'rectangle' = 'triangle';
```

Comment:
```suggestion
  shape: Shape = Shape.TRIANGLE;
```
потім отак перевикористай